CHANGELOG.md

```diff
@@ -1,5 +1,282 @@
+- Updated the links for the discussions and changelog ([daf37a1fce](https://github.com/facebook/react-native/commit/daf37a1fce43403e6320e1e3023e86fd1b970bdf) by [@MikeyAlmighty](https://github.com/MikeyAlmighty))
+- Upgraded to `prettier@^2.4.1`. ([4e587a1b7d](https://github.com/facebook/react-native/commit/4e587a1b7d5c3cd7efd5382e85fb4fcb935b2979) by [@yungsters](https://github.com/yungsters))
+- React Native sync for revisions afcb9cd...3fcd81d ([e161598c40](https://github.com/facebook/react-native/commit/e161598c404f2f585515e528c8980e9f269385ce) by [@ShikaSD](https://github.com/ShikaSD))
+- Upgraded react-devtools-core dependency to 4.21.0 ([18f62544c6](https://github.com/facebook/react-native/commit/18f62544c6f0f2459a9da325c6c843b49ebb23f0) by [@jstejada](https://github.com/jstejada))
+- XMLHttpRequest.getAllResponseHeaders() now returns headers with names lowercased and sorted in ascending order, as per specification ([b2415c4866](https://github.com/facebook/react-native/commit/b2415c48669391ee1ab7c6450748c4d91097a666) by [@ascherkus](https://github.com/ascherkus))
+- Upgraded react-devtools-core dependency to 4.20.2 ([016aef612c](https://github.com/facebook/react-native/commit/016aef612c254d3a815825f48886667254ca9253) by [@jstejada](https://github.com/jstejada))
+- Bump react-native-codegen to 0.0.9 ([e3a71b019f](https://github.com/facebook/react-native/commit/e3a71b019fa78e2b4b3454ccc59ea9c8cc543b29) by [@cortinico](https://github.com/cortinico))
+- Accessing `Image.propTypes`, `Text.propTypes`, `TextInput.propTypes`, `ColorPropType`, `EdgeInsetsPropType`, `PointPropType`, or `ViewPropTypes` now emits a deprecation warning. ([3f629049ba](https://github.com/facebook/react-native/commit/3f629049ba9773793978cf9093c7a71af15e3e8d) by [@yungsters](https://github.com/yungsters))
+- Upgraded react-devtools-core dependency to 4.20.1 ([288ca22583](https://github.com/facebook/react-native/commit/288ca225839a6f49a30a8a0fb0d69abbbac8b13e) by [@jstejada](https://github.com/jstejada))
+- Upgraded react-devtools-core dependency to 4.20.0 ([b5b1fa5f13](https://github.com/facebook/react-native/commit/b5b1fa5f1388f5614f6d443bcb24ed0c371c4585) by [@jstejada](https://github.com/jstejada))
+
+#### Android specific
+
+- Add allowsEdgeAntialiasing on views with rotations or skew tr… ([e6a3410afe](https://github.com/facebook/react-native/commit/e6a3410afe7d9a4cecf3db0a95503d2ff05bb862))
+- Bump Kotlin version to 1.6.10 ([d0f0234656](https://github.com/facebook/react-native/commit/d0f0234656dc981b422d1e9aa0885afd5fd29879) by [@AKB48](https://github.com/AKB48))
+- Bump Soloader to 0.10.3 ([f45889ef95](https://github.com/facebook/react-native/commit/f45889ef95ec694520e91b0032e591a087e088e5) by [@osartun](https://github.com/osartun))
+- Use side-by-side NDK for Android ([bd7caa64f5](https://github.com/facebook/react-native/commit/bd7caa64f5d6ee5ea9484e92c3629c9ce711f73c) by [@cortinico](https://github.com/cortinico))
```

Comment:
maybe this is also worth surfacing/highlighting? Kind of like "hey, you can uninstall any custom version of NDK that you have and just rely on what Android Studio installs for you"?